lms/djangoapps/commerce/views.py

```diff
@@ -123,3 +128,10 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         except ApiError as err:
             # The API will handle logging of the error.
             return InternalRequestErrorResponse(err.message)
+
+
+@cache_page(84600)
```

Comment:
I wondered about that. We don't seem to have a setting for default timeout. Decreasing to 30 minutes.